runtime/bcverify/rtverify.c

```diff
@@ -1092,6 +1092,14 @@ verifyBytecodes (J9BytecodeVerificationData * verifyData)
 			for (i = 0; i < popCount; i++) {
 				if (type1 == BCV_GENERIC_OBJECT) {
 					type = POP;
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+					if ((J9_ARE_ANY_BITS_SET(type, BCV_SPECIAL_INIT) || J9_ARE_ANY_BITS_SET(type, BCV_SPECIAL_NEW)) &&
+							((bc == JBifacmpeq) || (bc == JBifacmpne) || (bc == JBifnonnull) || (bc == JBifnull))) {
```

Comment:
Please add the final `) {` to the next line. The indentation should be in line with the if statement. Also flip the order of bc and the bytecode to follow convention in J9.